### ISSUE TITLE
Rule `todos` applied to `where` keyword.

### ISSUE THREAD

**Issue by QF5690** [nicklockwood/SwiftFormat#286]
I have code with long "where" expression similiar to this:
```
protocol Foo { associatedtype Baz }
protocol Bar { associatedtype Baz }

func foo<A, B, C>(a: A, b: B)
where A: Foo, A.Baz: Bar, A.Baz == C, C.Baz == String, B: Bar, B.Baz == A {
  print(a, b)
}
```
When I run swiftformat on it (indent 2), todos rule is applied, and result is:
```
func foo<A, B, C>(a: A, b: B)
  where A: Foo, A.Baz: Bar, A.Baz == C, C.Baz == String, B: Bar, B.Baz == A {
  print(a, b)
}
```
- It seems that applying todos rule is wrong here
- "where" blends visually with function body (although there is a chance that other people will prefer that)

Is it possible to make a new rule for that kind of cases?

**Comment by nicklockwood** [comment]
@QF5690 I’m not sure what the connection is to the `todos` rule? Does this indent go away if you disable the `todos` rule? That would be hard for me to explain if so.

This is the correct/expected behavior as implemented by the `indent` rule (wrapped lines of code are indented), however there is already a feature request to make the indent level for wrapped lines (aka “continuations”) configurable: https://github.com/nicklockwood/SwiftFormat/issues/191

**Comment by QF5690** [comment]
Hmm, it seems that there is a bug in telling that rule was applied:
```
$ swiftformat --indent 2 --verbose test/main.swift
running swiftformat...
formatting test/main.swift
 -- rules applied: todos
writing test/main.swift

swiftformat completed. 1/1 files updated in 0.01s
$ git reset HEAD --hard > /dev/null
$ swiftformat --indent 2 --verbose --disable indent test/main.swift
running swiftformat...
formatting test/main.swift
 -- no changes

swiftformat completed. 0/1 files updated in 0.01s
```

**Comment by nicklockwood** [comment]
@QF5690 oh, interesting. Can you send me your `main.swift` file? I'll try to replicate the issue.

**Comment by QF5690** [comment]
It is identical to code I supplied [above](url
https://github.com/nicklockwood/SwiftFormat/issues/286#issue-345860070), my version 0.34.0

**Comment by nicklockwood** [comment]
@QF5690 OK, I found the bug. Thanks!

**Comment by nicklockwood** [comment]
@QF5690 OK, the incorrect labelling of `todos` is fixed in version 0.34.1, but the only way to change the behavior you originally asked about is to disable the `indent` rule.

I'm going to close this ticket for now, but feel free to comment on https://github.com/nicklockwood/SwiftFormat/issues/191 if you have additional requirements.
